extension/app/scripts.babel/background.js

```diff
@@ -17,12 +17,18 @@
 
 'use strict';
 
-import {runPwaAudits} from './pwa-check.js';
+const ExtensionProtocol = require('../../../helpers/extension/driver.js');
```

Comment:
ah, yeah. At some point we should probably settle on an `import` or `require` style for extension-only code